addons/grenades/CfgAmmo.hpp

```diff
@@ -119,6 +119,13 @@ class CfgAmmo {
         effectsSmoke = "ACE_M84FlashbangEffect";
         whistleDist = 0;
     };
+    
+    class ACE_G_CTS9: ACE_G_M84 {
```

Comment:
It probably makes more sense to do that in #6848.

Reply:
If someone already changes the M84, but changing the inheritance you would break it even more.

Reply:
jonpas pointed out on Slack that very few people would want to use our pre-made class as it would introduce a direct dependency on ACE for their mods. In combination with what commy2 is saying, there is little point to a base class.

So, the question is: should we copy and paste the whole `ACE_G_M84` class here or inherit from it? Copying the whole class is probably safer, but this is your PR, dedmen, so I'll let you decide :)

Reply:
The base class would be just to make it easy for us. Not for others to inherit from.
I don't like code duplication so I did it like this.